AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+# AzTS Data Table
+
+
+## **Overview**
+
+Azure Tenant Security Solution centrally scans subscriptions and gets control scan results, processed RBAC data, resource inventory data, subscription metadata, etc., and pushes all the data into a blob storage of a central storage account.
```

Comment:
Would it be more appropriate to say "produces" control scan results? Since we're not getting it from somewhere but actually producing it?

Reply:
done